src/components/ScholarshipForm.jsx

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import InputLabel from '@material-ui/core/InputLabel';
+import MenuItem from '@material-ui/core/MenuItem';
+import FormControl from '@material-ui/core/FormControl';
+import Select from '@material-ui/core/Select';
+
+const useStyles = makeStyles((theme) => ({
+  formControl: {
+    margin: theme.spacing(1),
+    minWidth: 300,
+  },
+  selectEmpty: {
+    marginTop: theme.spacing(2),
+  },
+}));
+
+export default function ScholarshipForm() {
```

Comment:
Hmm, this is nice but is not strictly relevant for the Stepper. Perhaps create another branch and store these changes there?

Reply:
Yeah let's have this be a separate page. See #65. Take it on if you'd like to work on it

Reply:
This file was not created on a separate branch but I have it on my local computer, I will take a look at #65 and see if I can take on that PR

Reply:
Nvm Edwin I believe took care of PR #65, ill resolve this comment